tap/output.go

```diff
@@ -113,3 +70,17 @@ func (o *OutputLogger) Log(op *Output) error {
 
 	return nil
 }
+
+func (o *OutputLogger) CataLog(catalog *Catalog) error {
```

Comment:
```suggestion
func (o *OutputLogger) Catalog(catalog *Catalog) error {
```
..?

Reply:
This was a terrible joke from me - it's a catalog log 😢